drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -259,9 +259,12 @@ def label(key)
       end
     end
 
+    # More than one person in the household,
+    # and one household member must be less than 18
     private def custodial_parent_query
-      'jsonb_array_length(household_ages) > 1' +
-        'AND EXISTS(SELECT jsonb_array_elements(household_ages) AS age WHERE age < 18)'
+      query = ' jsonb_array_length(household_ages) > 1 '
+      query += " AND ma_yya_report_clients.id in ( SELECT ages.id FROM ( SELECT id, translate(household_ages::text, '[]', '{}')::integer [] AS h_ages FROM ma_yya_report_clients) ages WHERE 18 > ANY (h_ages) )"
```

Comment:
Wow, nice. I was messing with the `jsonb_array_elements` for a bit when finding this issue and was super confused.

Reply:
@awisema helped, took some serious head banging.